test/e2e/e2e.go

```diff
@@ -72,7 +72,10 @@ func setupProviderConfig() error {
 			managedZones = []string{zone}
 		}
 
-		gceAlphaFeatureGate, err := gcecloud.NewAlphaFeatureGate([]string{gcecloud.AlphaFeatureNetworkEndpointGroup})
+		gceAlphaFeatureGate, err := gcecloud.NewAlphaFeatureGate([]string{
```

Comment:
If you intend to submit this as alpha, you will also have to whitelist the alpha test project to be able to use the alpha disk api

Reply:
Yep, I intend to submit this as beta, but will do this for the regular test project

Reply:
For beta you don't need to whitelist right?

Reply:
We do